### DISCUSSION TITLE
No module named 'nonebot.adapters.onebot'

### DISCUSSION THREAD

**Discussion by QinCaii** [nonebot/discussions#161] [问答] [answered]
# python bot.py

Traceback (most recent call last):
  File "/root/Bot/bot.py", line 5, in <module>
    from nonebot.adapters.onebot.v11 import Adapter as ONEBOT_V11Adapter
ModuleNotFoundError: No module named 'nonebot.adapters.onebot'

我正在试着跟着教程把Windows上可以正常用的bot迁移到Linux上
但是在执行bot.py时出现了问题

**Comment by synodriver** [comment, marked as answer]
```pip install nonebot-adapter-onebot```
